new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -187,4 +187,18 @@ public static boolean doesImageExist(String searchString) {
   public static boolean isHelmReleaseDeployed(String releaseName, String namespace) {
     return Helm.isReleaseDeployed(releaseName, namespace);
   }
+
+  /**
+   * Check if a persistent volume is in a given state.
+   *
+   * @param name name of the persistent volume
+   * @param state state of the persistent volume to match
```

Comment:
desired state of the ...

Reply:
Since I removed the pvc and pv creation this assertion is no loner needed.